openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -103,12 +102,12 @@ <h4 class="question__title ui-slidable__control">
 
                     {% if peer_assessments %}
                         <form class="submission__feedback submission__feedback step__content__section ui-slidable__container is--showing" method="post">
-                            <h3 class="submission__feedback__title ui-slidable__control">
+                            <h4 class="submission__feedback__title ui-slidable__control">
```

Comment:
How did you decide h4 here (it seems like most of the headers are decreasing 2 levels)?